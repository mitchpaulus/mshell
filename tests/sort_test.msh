"# Basic sort test" wl
[hello 1 'c' 'A'] sort uw

"# Unique sort test" wl
[z y 'x' y z] sortu uw
