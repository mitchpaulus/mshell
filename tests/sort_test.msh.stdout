# Basic sort test
1
A
c
hello
# Unique sort test
x
y
z
